app/api/routes.py

```diff
@@ -61,15 +61,21 @@ def random_plant():
 def search_plants():
     search = request.args['q']
     plants = db.session.query(Plant).filter(Plant.plant_type.ilike('%{}%'.format(search))).all()
-    results = []
 
-    for plant in plants:
-        obj = {
-            'id': plant.id,
-            'plant_type': plant.plant_type,
-            'plant_image': plant.image,
+    if len(plants) == 0:
+        results = {
+            'image': 'error image',
```

Comment:
Nice sad path consideration here! We'll have to input a picture here later as well